### PR TITLE
Remove 'using namespace std;' from the qglviewer files

### PR DESCRIPTION
**Pull request by afabri**
## Summary of Changes

Poses a problem for the ImagIO files as `byte` becomes ambiguous with VS2017and `/std:c++latest`

## Release Management

* Affected package(s):  Polyhedron demo

### CODE REVIEW

File: GraphicsView/include/CGAL/Qt/camera_impl.h [hunk 1]
```diff
@@ -36,7 +36,6 @@
 #include <CGAL/Qt/domUtils.h>
 #include <CGAL/Qt/keyFrameInterpolator.h>
 
-using namespace std;
 using namespace CGAL::qglviewer;
```
Comment (lrineau) [inline GraphicsView/include/CGAL/Qt/camera_impl.h]: Actually, I also think that that
```C++
using namespace CGAL::qglviewer;
```
All those `_impl` files should be in the `CGAL::qglviewer` namespace.

**Comment by afabri** [comment]
I leave that for @maxGimeno

**Comment by lrineau** [comment]
TODO for @maxGimeno: remove all those 
```c++
using namespace CGAL::qglviewer;
```

**Comment by lrineau** [comment]
@maxGimeno Something simple on your TODO list: remove the `using namespace CGAL::qglviewer;` from the "_impl" files of QGLViewer.

**Comment by maxGimeno** [comment]
done.

**Comment by lrineau** [comment]
Successfully tested in [CGAL-4.13-Ic-37][*], and then merged.

  [*]: https://cgal.geometryfactory.com/CGAL/testsuite/results-4.13-Ic-37.shtml

**Pull request merged**